src/content/en/updates/2017/03/background_tabs.md

```diff
@@ -0,0 +1,70 @@
+# Background tabs in Chrome 57.
+
+Background tabs can have a dramatic negative effect on browser performance, especially on battery life. To mitigate this, Chrome have been placing various restrictions on background tabs for last several years. Recently there’s been a number of efforts to make further improvements to the situation, and this document gives an overview of what Chrome policy is. This document focuses on describing current policies in Chrome 57. Long-term strategy and further plans can be found in [this document](https://docs.google.com/document/d/1FCcMdUDl25RCnzfAbKOur5yVACCjS1EV4PUvlPRQ-9w/edit).
+
+## Optimising an application for background
+
+Web developers should be aware that users often have a lot of tabs open in the background and it have a serious effect on power usage and battery life. Work in background should be kept to minimum unless it’s absolutely necessary to provide a particular user experience. [Page visibility API](https://developer.mozilla.org/en-US/docs/Web/Events/visibilitychange) should be used to detect when page is backgrounded and suspend all necessary work like visual updates.
+
+[For some sites](https://twitter.com/cryptowat_ch/status/817502626896089090), this simple optimisation can reduce CPU usage by 75%: 
+
+```
+var doVisualUpdates = true;
+
+document.addEventListener('visibilitychange', function(){
+
+  doVisualUpdates = !document.hidden;
+
+});
+
+function update() {
+
+  if (!doVisualUpdates) {
+
+    return;
+
+  }
+
+  doStuff();
}
+```
+
+## Policies
+
+### requestAnimationFrame
+
+Per [the spec](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame), Chrome does not run requestAnimationFrame callback when page is in background. This behavior was in place since 2011.
```

Comment:
Change 'Chrome does not run requestAnimationFrame callback' to 'Chrome does not call `requestAnimationFrame()`.
Change 'This behavior was in place since 2011' to 'This behavior has been in place since 2011'.